### ISSUE TITLE
Add CI Jobs for Conda

### ISSUE THREAD

**Issue by Zethson** [mlf-core/mlf-core#269]
Currently we are only testing our runs with Docker on Github Actions. This is easier, since we know that CUDA is setup there and things will just sail smoothly.

I would like to evaluate whether runs with Conda also work on GA and if so to add a job for the create_mlflow_x workflows.

@Emiller88 this issue could be one for you?

**Comment by Zethson** [comment]
So TLDR - won't work, right @Emiller88 ?
The I would close this for now and reopen when we have GPU CI somewhen

**Comment by Emiller88** [comment]
Yep I think you're right. There won't be the necessary stuff on the GitHub runners to use conda.
